lib/errors.ts

```diff
@@ -122,6 +122,22 @@ const messages: {
 
 	EACCES: (e) => messages.EPERM(e),
 
+	BalenaSettingsPermissionError: () => stripIndent`
+		You don't have sufficient privileges to access balena settings.
+
+		Balena stores the session token in ${
+			os.platform() === 'win32' ? 'C:\\Users\\<user>\\_balena' : '$HOME/.balena'
```

Comment:
These 2 lines could be deleted if my suggestion in the other comment is adopted (using `${dataDirectory}` in the first line of the error message). These hardcoded strings for Windows and Linux/macOS could be used as default values for `dataDirectory` if its value cannot be fetched.